src/i18n/messages/all.de.json

```diff
@@ -52,7 +52,10 @@
   "calendar": {
     "nextMonthAriaLabel": "Nächster Monat",
     "previousMonthAriaLabel": "Vorheriger Monat",
-    "todayAriaLabel": "Heute"
+    "todayAriaLabel": "Heute",
+    "i18nStrings.nextYearAriaLabel": "Nächstes Jahr",
+    "i18nStrings.previousYearAriaLabel": "Vorheriges Jahr",
+    "i18nStrings.currentMonthAriaLabel": "Kosten des aktuellen Monats"
```

Comment:
Okay this is kinda hilarious, but this is not translated correctly 😂  
This translates to "Costs of the current month". It should just say "Aktueller Monat".

Reply:
Yes, we were discussing this also [here](https://github.com/cloudscape-design/components/pull/1972/files#r1492179604). This will definitely need to be fixed before merging.